server/monitor/src/main/java/org/apache/accumulo/monitor/rest/gc/GarbageCollectorResource.java

```diff
@@ -24,13 +24,12 @@
 import jakarta.ws.rs.Produces;
 import jakarta.ws.rs.core.MediaType;
 
-import org.apache.accumulo.core.gc.thrift.GCStatus;
 import org.apache.accumulo.monitor.Monitor;
 
 /**
  * GarbageCollector metrics
  *
- * @since 2.0.0
+ * @since 2.1.0
```

Comment:
You're changing it in 2.1, but it the `/gc` endpoint was still introduced in 2.0. These aren't exactly public API, so it shouldn't matter, but it is a little strange to change this to an incorrect value.

Reply:
I wasn't sure what to do cause I was changing the endpoint. They aren't public API but maybe we should treat them like API? I feel like we shouldn't be required since the Monitor is only an admin tool. I guess keeping the tag for when it was created is useful.